src/constants/wallets.ts

```diff
@@ -341,7 +341,7 @@ export type DydxAddress = `dydx${string}`;
 export const DYDX_CHAIN_INFO: Parameters<typeof suggestChain>[0] = {
   rpc: '13.59.4.93:26657',
   rest: '13.59.4.93:1317',
-  chainId: 'dydx-testnet-2',
+  chainId: 'dydx-testnet-3',
```

Comment:
does the rpc/rest ip match with this chainId?

Reply:
oops, good call